<launch>

    <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan">
        <remap from='image' to='/zed/zed_node/depth/depth_registered' />
        <remap from='/camera_info' to='/zed/zed_node/depth/camera_info' />
        
        <param name="scan_height" value="4" />
        <param name="scan_time" value="0.1" />
        <param name="output_frame_id" value="zed_left_camera_frame" />

    </node>

</launch>